qfdmo/models/action.py

```diff
@@ -74,7 +74,7 @@ def primary(self):
                 for key, value in DSFRColors.items()
                 if value.lower() == self.couleur.lower()
             ),
-            None,
+            "",
```

Comment:
Les chaînes vides me font toujours peur. Pourquoi ne veux-tu plus du `None` que tu avais avant? La raison d'un point de vue data engineering c'est qu'**il devient bcp plus difficile de gérer le cas vide** (None est unique, chaque type à sa propre expression du vide).

C'est un problème sérieux dans notre code qui à déjà infiltré la base de données:

`SELECT count(*) FROM qfdmo_acteur WHERE code_postal IS NULL` = 3398 entrées

`SELECT count(*) FROM qfdmo_acteur WHERE code_postal IS NULL OR code_postal = ''` = 5229 entrées

Reply:
Il y a une convention - pas forcément claire - qui consiste à retourner des string vides dans les templates plutôt que `None` qui dans certains cas peut être stringifié et affiché comme `"None"`. 

Ici, nos properties `primary` et `secondary` ne sont utilisées qu'à des fins d'affichage. 
Ce qui me fait me dire qu'elles mériteraient un nommage adéquat. 
Ça rejoint la discussions que tu avais ouvert concernant la distinction modeling / rendering (#1016).  
Une fois que la vague de refacto data sera passée, je serais partant pour qu'on se cale un moment avec Nico et toi pour échanger là dessus et définir une approche qui va à tout le monde.